modin/core/execution/ray/common/utils.py

```diff
@@ -219,11 +167,15 @@ def initialize_ray(
             if not GPU_MANAGERS:
                 for i in range(GpuCount.get()):
                     GPU_MANAGERS.append(GPUManager.remote(i))
-    _move_stdlib_ahead_of_site_packages()
-    ray.worker.global_worker.run_function_on_all_workers(
-        _move_stdlib_ahead_of_site_packages
-    )
-    ray.worker.global_worker.run_function_on_all_workers(_import_pandas)
+    else:  # ray is already initialized, check runtime env config
+        env_vars = ray.get_runtime_context().runtime_env.get("env_vars", {})
```

Comment:
This only checks that the head nodes env vars are set correctly, right? Don't we need to set the environment variable on all of the workers/nodes, and if so, shouldn't we be checking that the env vars are set correctly on all of the workers? We could probably do that using the ray function that runs on all workers.

Reply:
No, `.runtime_env` is the thing which _tells_ Ray which environment variables to set for its workers. It should set these variables for the workers automatically.